README.md

```diff
@@ -114,49 +114,105 @@ The Vonage Video API Reference App for React is currently supported on the lates
+<details open>
+<summary>Applications dashboard view</summary>
+<img src="./docs/assets/readme/1-dashboard-applications.png" alt="Applications dashboard" style="max-width: 100%; height: auto;" />
+</details>
+
+If you don’t already have an application, create a new one:
+
+<details open>
+<summary>Create new app</summary>
+<img src="./docs/assets/readme/2-create-app.png" alt="Create app button" style="max-width: 100%; height: auto;" />
+</details>
+
+During the setup process, make sure to:
+
+- Provide a name for your application.
+- Generate and download the public and private keys.
+- Enable **Video** capabilities.
+
+Refer to the following image for visual guidance:
+
+<details open>
+<summary>Configuring a new app</summary>
+<img src="./docs/assets/readme/3-create-app-form.png" alt="Create app form" style="max-width: 100%; height: auto;" />
+</details>
+
+### 3. Environment Variable
+
+In the root project directory, create the environment files by running:
+
+``` bash
+cp backend/.env.example backend/.env && cp frontend/.env.example frontend/.env
 ```
-[Click here](frontend/.env.example) to learn more about config variables used in the frontend.
 
-### Running the project
+Then, open **backend/.env** and fill in the required configuration:
 
-#### Installing dependencies
+- **VONAGE_APP_ID** – This is the ID of your Vonage application. You can find it on the [Applications page](https://dashboard.vonage.com/applications).
+- **VONAGE_PRIVATE_KEY** – If you've already generated a private key, use that. Otherwise, use the key you downloaded when creating the app.
 
-```console
+### 4. Running the Project
+
+#### Install Dependencies
+
+``` bash
 yarn
 ```
-This command installs all appropriate dependencies for the project. If you would like more information on the packages we use, please refer to the [Dependencies](./docs/DEPENDENCIES.md) document.
 
-#### Dev mode
+This command installs all necessary dependencies. For details about the packages used, refer to [Dependencies](./docs/DEPENDENCIES.md).
 
-```console
+#### Start in Development Mode
+
+``` bash
 yarn dev
 ```
 
-This command builds and watches both the backend server (:3345) and frontend vite dev server (:5173)
-You should now see the app running at [http://localhost:5173/](http://localhost:5173/)
+This starts both the backend server (port **3345**) and the frontend Vite dev server (port **5173**). You can now access the app at [http://localhost:5173](http://localhost:5173).
 
-#### Production mode
+### 5. Testing on Multiple Devices
 
-This command builds a production bundle of the app frontend and `cp`s it to the backend to be served by the express server.
+To test the video API across multiple devices on your local network, you can use **ngrok** to expose your frontend publicly.
 
-```console
-yarn start
+#### Steps:
+
+1. Create an account at [ngrok](https://dashboard.ngrok.com/signup) if you haven’t already (The free tier allows you to have one public tunnel).
```

Comment:
I'd avoid mentioning what `ngrok` allows on their free tier as that can change at any time. I'd also like to hear from other reviewers if this section is needed.